src/product-service/handlers/getProductsById.ts

```diff
@@ -0,0 +1,32 @@
+import products from '../data.json';
+import { APIGatewayProxyEvent } from 'aws-lambda';
+export const getProductsById = async (event: APIGatewayProxyEvent) => {
+  const productId = event.pathParameters?.productId;
+  if (!productId) {
```

Comment:
Please add empty lines before ifs and before return - it contributes readability